tchannel/testing/vcr/patch.py

```diff
@@ -82,27 +82,34 @@ def send(self, arg1, arg2, arg3,
         headers = headers or {}
         headers.setdefault('as', self.arg_scheme)
 
-        vcr_request = VCRProxy.Request(
+        vcr_request = proxy.Request(
             serviceName=self.service.encode('utf-8'),
             hostPort=self.hostport,
             knownPeers=self.original_tchannel.peers.hosts,
             endpoint=endpoint,
             headers=(yield read_full(arg2)),
             body=(yield read_full(arg3)),
-            argScheme=getattr(VCRProxy.ArgScheme, self.arg_scheme.upper()),
+            argScheme=getattr(proxy.module.ArgScheme, self.arg_scheme.upper()),
```

Comment:
You don't need to do `proxy.module.foo`. You can just do `proxy.foo`. `.module` is not a public API. I'll add a `_` later to make that obvious.